### PR TITLE
final.py: Remove unused imports  

### LINKED ISSUE #585: final.py: Unused imports
Pyflakes error report =>
```
./final.py:3: 'gensim.parsing.preprocessing.STOPWORDS' imported but unused
./final.py:4: 'gensim.utils.simple_preprocess' imported but unused
./final.py:6: 'nltk' imported but unused
```

### PR DESCRIPTION
**Pull request by Pawank221**
Closes #585

# Reviewers Checklist

- [ ] Appropriate logging is done.
- [ ] Appropriate error responses.
- [ ] Handle every possible exception.
- [ ] Make sure there is a docstring in the command functions. Hint: Lookout for
  `botcmd` and `re_botcmd` decorators.
- [ ] See that 100% coverage is there.
- [ ] See to it that mocking is not done where it is not necessary.

### CODE REVIEW

File: answers/final.py [hunk 1]
```diff
@@ -1,9 +1,6 @@
 from collections import OrderedDict, Counter
 
-from gensim.parsing.preprocessing import STOPWORDS
-from gensim.utils import simple_preprocess
 import networkx
-import nltk
```
Comment (jayvdb) [inline answers/final.py]: if you like, you can also remove it from https://github.com/coala/corobo/blob/master/answers/requirements.txt

**Comment by MacBox7** [comment]
@Pawank221 please reply and fix this soon otherwise you will be unassigned. For any help ask on coala group.

**Comment by jayvdb** [comment]
@gitmate-bot rebase

**Comment by jayvdb** [comment]
ack https://github.com/coala/corobo/pull/589/commits/fcfbf85053f57c3e858657ca715280004607f5a7

**Comment by jayvdb** [comment]
@gitmate-bot ff

File: answers/final.py (hunk 1 above)
Comment (jayvdb) [reply answers/final.py]: Created new issue about that https://github.com/coala/corobo/issues/593 , for another newcomer to complete

**Pull request merged**